aten/src/ATen/functorch/BatchRulesLoss.cpp

```diff
@@ -46,6 +49,45 @@ mse_loss_batch_rule(const at::Tensor& self, optional<int64_t> self_bdim, const a
   TORCH_INTERNAL_ASSERT(false);
 };
 
+std::tuple<at::Tensor,optional<int64_t>>
+mse_loss_batch_rule(const at::Tensor& self, optional<int64_t> self_bdim, const at::Tensor& target,
+          optional<int64_t> target_bdim, int64_t reduction) {
+  return loss_batch_rule_helper(self, self_bdim, target, target_bdim,
+                                reduction, [](const at::Tensor& a, const at::Tensor& b, int64_t red) {
```

Comment:
I think it would be nice to rename `a` -> `self` , `b` -> `target` and `red` -> `reduction`.